GetandTake/Program.cs

```diff
@@ -1,14 +1,28 @@
 using AutoMapper;
-using GetandTake.Configuration;
-using GetandTake.Services.AutoMapper;
-using static System.Net.Mime.MediaTypeNames;
+using GetandTake.Business.Services.AutoMapper;
+using GetandTake.Configuration.Extensions;
+using GetandTake.Configuration.Settings;
+using GetandTake.Core.DependencyResolvers;
+using GetandTake.Core.Extensions;
+using GetandTake.Core.Filters;
+using GetandTake.Core.Utilities.IoC;
+using GetandTake.Middlewares;
 
 var builder = WebApplication.CreateBuilder(args);
 builder.RegisterLogging();
 // Add services to the container.
 builder.Services
     .AddRazorPages()
+    //.AddMvcOptions(options =>
+    //{
+    //    options.Filters.Add(typeof(LogActionFilterAttribute));
+    //})
```

Comment:
Is it necessary to comment this section?

Reply:
I forgot to un-comment that part. :)